### DISCUSSION TITLE
Could we have a way to change how Resources are named in the docs Schema section?

### DISCUSSION THREAD

**Discussion by maxechendu** [dedoc/scramble#545] [Ideas]
Not sure if this is already a feature, but can't find it in the scramble docs, and can't seem to achieve it.

Whenever resources are used, they're visible in the **SCHEMA** section as their class name. Is there a way to override this, and display something else in the docs?

I've tried adding a description comment above the class, like so:

```php
/**
 * Basic Schema
 */
class BasicResource extends JsonResource
{
    // ...
}
```

Or this:

```php
/**
 * @tags Basic Schema
 */
class BasicResource extends JsonResource
{
    // ...
}
```

Nothing works, so assuming this feature isn't in?

Here an image of the section below:

<img width="309" alt="image" src="https://github.com/user-attachments/assets/5c6e2150-06ff-4fe4-a6a8-c016ddb77593">

**Comment by romalytvynenko** [comment]
@maxechendu yes! 

Please track this issue: https://github.com/dedoc/scramble/issues/479

**Reply by maxechendu** [reply]
Thank you 😃
